### PR TITLE
Upgrade 2.x branch to newer gdal-warp

### LINKED ISSUE #176: Update v2 branch with latest gdal-warp-bindings
Currently having to use `dependencyOverrides` in RasterFrames and all downstream projects.

### PR DESCRIPTION
**Pull request by metasim**
Closes #176

@pomadchin Can you publish a new version of this to bintray?

### CODE REVIEW

**Comment by metasim** [comment]
Actually, hold off on this... getting a core dump.

[hs_err_pid66771.log.zip](https://github.com/geotrellis/geotrellis-contrib/files/3232820/hs_err_pid66771.log.zip)

File: project/Dependencies.scala [hunk 1]
```diff
@@ -23,7 +23,7 @@ object Version {
   val geotrellis     = "2.3.0"
   val geotrellisGdal = "0.18.5"
   val gdal           = Properties.envOrElse("GDAL_VERSION", "2.4.0")
-  val gdalWarp       = "33.e53ec75"
+  val gdalWarp       = "33.29f906d"
```
Comment (pomadchin) [inline project/Dependencies.scala]: Probably it should be `33.5523882`

**Comment by pomadchin** [comment]
@metasim wondering how did you get this? Since I've lunched dozens of ingest jobs (with `33.5523882` version), see the results here: https://github.com/pomadchin/vlm-performance

**Comment by metasim** [comment]
I got it in a RasterFrames integration test.... but let me try with the version you reference.

**Comment by pomadchin** [comment]
Hey @metasim what is the status of this PR?

**Comment by metasim** [comment]
Need to do more testing when I have time.

**Comment by pomadchin** [comment]
This PR bumps 2.x branch gdal-warp bindings version up to `33.58d4965`

**Comment by pomadchin** [comment]
@metasim do you want me to publish a new release?

**Comment by metasim** [comment]
@pomadchin That would be super fantastic! Thanks!

**Comment by pomadchin** [comment]
@metasim will do it today and notify you :+1:

**Comment by metasim** [comment]
@pomadchin Did this get published?

**Comment by pomadchin** [comment]
@metasim ah sorry; totally forgot; will do it by the end of the day!

**Pull request merged**